apps/backoffice-v2/src/pages/Entities/components/CaseGeneration/components/CaseGenerationForm/utils/transformErrors.ts

```diff
@@ -0,0 +1,48 @@
+import { AnyObject } from '@ballerine/ui';
+import { RJSFValidationError } from '@rjsf/utils';
+
+export const transformErrors = (errors: RJSFValidationError[]): RJSFValidationError[] => {
+  return errors.map(error => {
+    console.log('error', error);
```

Comment:
Is this log supposed to be removed? If not, let's use `terminal` here or `console.error` like we do with Zod.

Reply:
Why not log `errors` or the output of `transformErrors`?